src/main/java/randoop/reflection/OperationModel.java

```diff
@@ -365,6 +405,15 @@ public ContractSet getContracts() {
     return contracts;
   }
 
+  /**
+   * Returns the omit methods predicate associated with this operation model.
```

Comment:
Here is another place that I didn't find the comments helpful.  Are these methods that the user has specified should not be called?  If so, say that.  Otherwise, say what it is.